website/content/en/status/report-2023-01-2023-03/ufs_snapshots.adoc

```diff
@@ -2,7 +2,8 @@
 
 Contact: Marshall Kirk McKusick <mckusick@freebsd.org>
 
-The ability to make UFS/FFS filesystem snapshots when running with journaled soft updates and using them for doing background dumps on a live filesystem is available in the 13.2 release.
+The ability to make UFS/FFS filesystem snapshots when running with journaled soft updates, and using them for doing background dumps on a live filesystem, was merged to `releng/13.2` during the first quarter of 2023, and lands in FreeBSD 13.2-RELEASE.
```

Comment:
Why `lands`? Shouldn't it be `has landed`?

Reply:
I also used past tense [in an earlier draft](https://github.com/freebsd/freebsd-doc/pull/160/commits/cf4d5707cc0e7627a41cac6f2037e4f48a6f5cbe#diff-ab18c7b451449778d20fa3d5a180871c4ab489d0b4006bb0a11a763a31f99823R5). In doing so, I lost sight (in my mind's eye) of the main heading, and HTML title: 

> # FreeBSD Status Report First Quarter 2023

From <https://github.com/freebsd/freebsd-doc/commit/569297cf1d887f469ea60306ce05396db25934e0>: 

> … Change the wording a little, as if reviewed at the end of – still
within – the quarter. Not so much change that it appears to be a late
submission.